File: program/program/find-surface-area-of-a-cylinder/FindSurfaceAreaOfACylinder.cpp
#define _USE_MATH_DEFINES

#include <iostream>
#include <cmath>

int main()
{

    float height, radius, surfaceAreaOfCylinder;

    std::cout << "enter height of cylinder" << std::endl;
    std::cin >> height;

    std::cout << "enter radius of cylinder" << std::endl;
    std::cin >> radius;

    surfaceAreaOfCylinder = 2 * M_PI * radius * (radius + height);

    std::cout << "surface area of this cylinder is " << surfaceAreaOfCylinder;

    return 0;
}

File: program/program/find-surface-area-of-a-cylinder/FindSurfaceAreaOfACylinder.fs
let surfaceArea radius height : float =
   let pi = 3.14159
   2.0 * pi * radius * (radius+height)

printfn "Enter the radius :"
let radius = float (System.Console.ReadLine())
printfn "Enter the height :"
let height = float (System.Console.ReadLine())
let cylinder = surfaceArea radius height

printfn "Surface area of cylinder : %f" cylinder
